packages/math-input/src/data/key-configs.ts

```diff
@@ -254,22 +257,24 @@ const KeyConfigs: any = {
         // I18N: A label for a button that will dismiss/hide a keypad.
         ariaLabel: i18n._("Dismiss"),
     },
-};
 
-// Add in any multi-function buttons. By default, these keys will mix in any
-// configuration settings from their default child key (i.e., the first key in
-// the `childKeyIds` array).
-// TODO(charlie): Make the multi-function button's long-press interaction
-// accessible.
-// NOTE(kevinb): This is only used in the mobile native app.
-KeyConfigs[Keys.FRAC_MULTI] = {
-    childKeyIds: [Keys.FRAC_INCLUSIVE, Keys.FRAC_EXCLUSIVE],
-};
+    // Add in any multi-function buttons. By default, these keys will mix in any
+    // configuration settings from their default child key (i.e., the first key in
+    // the `childKeyIds` array).
+    // TODO(charlie): Make the multi-function button's long-press interaction
+    // accessible.
+    // NOTE(kevinb): This is only used in the mobile native app.
+    [Keys.FRAC_MULTI]: {
+        childKeyIds: [Keys.FRAC_INCLUSIVE, Keys.FRAC_EXCLUSIVE],
+    },
+
+    // TODO(charlie): Use the numeral color for the 'Many' key.
+    [Keys.MANY]: {
+        type: KeyType.MANY,
+        // childKeyIds will be configured by the client.
+    },
 
-// TODO(charlie): Use the numeral color for the 'Many' key.
-KeyConfigs[Keys.MANY] = {
-    type: KeyType.MANY,
-    // childKeyIds will be configured by the client.
+    [Keys.PERIOD]: {},
```

Comment:
This looks new! Is it meant to be empty? It looks like all other keys have a type and an arialabel.

Reply:
I had to add something for every key to type this. This one had nothing defined previously.

Reply:
Isn't the key just typed to `string` though (`const KeyConfigs: {[key: string]: any}`)?

Reply:
Oh yeah, you're right. This was from my effort to type the key to Keys, but when a number button tried to access by `NUM_9` it'd blow up bc it's programmatically added to the KeyConfigs object keys instead of the enum